sc_rf_service/sc_rf_service.py

```diff
@@ -47,18 +55,45 @@ class CryomodulePV(PVGroup):
     cryoSumLatchB = pvproperty(value=0, name=":CRYOSUMB_LTCH",
                                dtype=ChannelType.ENUM,
                                enum_strings=("OK", "Faulted"))
-    beamVacuumLatchA = pvproperty(value=0, name=":BMLNVACA_LTCH",
-                                  dtype=ChannelType.ENUM,
-                                  enum_strings=("OK", "Faulted"))
-    beamVacuumLatchB = pvproperty(value=0, name=":BMLNVACB_LTCH",
-                                  dtype=ChannelType.ENUM,
-                                  enum_strings=("OK", "Faulted"))
-    couplerVacuumLatchA = pvproperty(value=0, name=":CPLRVACA_LTCH",
-                                     dtype=ChannelType.ENUM,
-                                     enum_strings=("OK", "Faulted"))
-    couplerVacuumLatchB = pvproperty(value=0, name=":CPLRVACB_LTCH",
-                                     dtype=ChannelType.ENUM,
-                                     enum_strings=("OK", "Faulted"))
+
+
+class RackPV_HWI(PVGroup):
+    hwi = pvproperty(value=0.0, name=":HWINITSUM", dtype=ChannelType.ENUM,
+                     enum_strings=("Ok", "HW Init running", "LLRF chassis problem"))
+
+    def __init__(self, device_name, change_callback, initial_values, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.device_name = device_name
+        self.element_name = initial_values[3]
+
+
+class RackPV_Vacuum(PVGroup):
+    beamLineVacuumA = pvproperty(value=0.0, name=":BMLNVACA_LTCH", dtype=ChannelType.ENUM,
+                                 enum_strings=("Ok", "Fault"))
+    beamLineVacuumB = pvproperty(value=0.0, name=":BMLNVACB_LTCH", dtype=ChannelType.ENUM,
+                                 enum_strings=("Ok", "Fault"))
+
+    def __init__(self, device_name, change_callback, initial_values, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.device_name = device_name
+        self.element_name = initial_values[3]
+
+
+class RackPV_couplerVacuum(PVGroup):
+    couplerVacuumA = pvproperty(value=0.0, name=":CPLRVACA_LTCH", dtype=ChannelType.ENUM,
+                                enum_strings=("Ok", "Fault"))
+    couplerVacuumB = pvproperty(value=0.0, name=":CPLRVACB_LTCH", dtype=ChannelType.ENUM,
+                                enum_strings=("Ok", "Fault"))
+
+    def __init__(self, device_name, change_callback, initial_values, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.device_name = device_name
+        self.element_name = initial_values[3]
+
+
+class WatcherPV(PVGroup):
+    heartbeat = pvproperty(value=0, name="SC_CAV_FAULT_HEARTBEAT",
+                           dtype=ChannelType.INT)
```

Comment:
We should also add `ALRM:SYS0:SC_CAV_FAULT:ALHBERR`

Reply:
Added!